tests/benchmarks/multicore/idle_pwm_loopback/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config REMOTE_BOARD
+	string "The board used for remote target"
```

Comment:
from where does this pattern of having a prompt here originates ?

The exact same construct was commented here: https://github.com/nrfconnect/sdk-nrf/pull/19018#discussion_r1853704191
and I would like to get this construct removed from the root source where people copy things.

Please adjust in similar fashion as describe in this comment: https://github.com/nrfconnect/sdk-nrf/pull/19018#discussion_r1853704191

Reply:
It's all over the place :smile:
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_adc/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_clock_control/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_comp/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_gpio/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_hpu_temp_meas/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_outside_of_main/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_spim/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_twim/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_uarte/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_usb/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_wdt/Kconfig.sysbuild#L10
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_with_pwm/Kconfig.sysbuild#L10

I think, this solution was proposed in the first idle_xxx test and then copied to many others.

Reply:
> This solution seems to have one advantage - it allows to select different remote core for the same 'build" core. For example:

and if that is a need, then I have a solution for that as well :wink: